src/combobox/combobox.component.ts

```diff
@@ -186,6 +185,30 @@ export class ComboBox implements OnChanges, AfterViewInit, AfterContentInit {
 	 * Value displayed if dropdown is in invalid state.
 	 */
 	@Input() invalidText: string | TemplateRef<any>;
+	/**
+	 * Value to display for accessibility purposes on the combobox control menu when closed
+	 */
+	@Input() openMenuAria = "open menu";
+	/**
+	 * Value to display for accessibility purposes on the combobox control menu when opened
+	 */
+	@Input() closeMenuAria = "close menu";
+	/**
+	 * Value to display on the clear selections icon, when multi is selected
+	 */
+	@Input() clearSelectionsTitle = "Clear all selected items";
+	/**
+	 * Value to display for accessibility purposes to clear selections, when multi is selected
+	 */
+	@Input() clearSelectionsAria = "Clear all selected items";
+	/**
+	 * Value to display on the clear the selected item icon, when single is selected
+	 */
+	@Input() clearSelectionTitle = "Clear selected item";
+	/**
+	 * Value to display for accessibility purposes on the clear the selected item icon, when single is selected
+	 */
+	@Input() clearSelectionAria = "Clear Selection";
```

Comment:
We should add these into translatable text, similar to the way the number component does this, check out https://github.com/IBM/carbon-components-angular/blob/9b1d712fcb9d269155dbe14e2c25c544183a865f/src/number-input/number.component.ts#L177-L198

Reply:
Let me know if it's how you wanted.

Reply:
Perfect!